README.rst

```diff
@@ -114,6 +116,11 @@ Documentation can be generated in several formats::
     epub : make epub
     html : make html
 
+Contributing to libdrizzle-redux
+--------------------------------
+
+Please refer to the guide for users of libraries at `Neptune Wiki`_.
```

Comment:
This is a bit thrown in the middle. What's Neptune about ? I am wondering where a function is used, is that where I should go ?